assets/src/edit-story/components/panels/panel/simplePanel.js

```diff
@@ -28,8 +28,8 @@ import PanelContent from './shared/content';
 
 function SimplePanel({ children, name, title, ...rest }) {
```

Comment:
`[no-unused-vars] 'rest' is defined but never used.`

Reply:
Ah, got it, this is from the conflict resolve. In this case, I should keep it.